README.md

```diff
@@ -3,406 +3,224 @@
-
-    // Obtain the text of the article header
-    let headerText = await articleHeader.innerText;
-});
-```
-
-For more information, see [Selecting Page Elements](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/).
-
-#### Assertions
-
-A functional test also should check the result of actions performed.
-For example, the article header on the "Thank you" page should address a user by the entered name.
-To check if the header is correct, you have to add an assertion to the test.
-
-The following test demonstrates how to use [build-in assertions](https://devexpress.github.io/testcafe/documentation/test-api/assertions/).
-
-```js
-import { Selector } from 'testcafe';
+Read the [Getting Started](https://devexpress.github.io/testcafe/documentation/getting-started/) page for more information.
 
-fixture `Getting Started`
-    .page('https://devexpress.github.io/testcafe/example');
+![Install TestCafe and Run a Test](https://raw.githubusercontent.com/DevExpress/testcafe/master/media/install-and-run-test.gif)
 
-test('My first test', async t => {
-    await t
-        .typeText('#developer-name', 'John Smith')
-        .click('#submit-button')
-        // Use the assertion to check if the actual header text is equal to the expected one
-        .expect(Selector('#article-header').innerText).eql('Thank you, John Smith!');
-});
-```
+<p align="center">
+<i>Running a sample test in Safari</i>
+</p>
 
 ## Documentation
 
-* [Test API](https://devexpress.github.io/testcafe/documentation/test-api/)
-* [Using TestCafe](https://devexpress.github.io/testcafe/documentation/using-testcafe/)
-* [Extending TestCafe](https://devexpress.github.io/testcafe/documentation/extending-testcafe/)
-* [Recipes](https://devexpress.github.io/testcafe/documentation/recipes/)
-
-## TestCafe Ecosystem
-
-The following plugins make your work with TestCafe easy and introduce new features.
+Go to our website for full [documentation](http://devexpress.github.io/testcafe/documentation/using-testcafe/) on TestCafe.
 
-### IDE Plugins
+## Community
 
-Run tests and view test results right from your favorite IDE.
+Follow us on [Twitter](https://twitter.com/DXTestCafe). We post TestCafe news and updates, several times a week.
 
-* [Visual Studio Code](https://github.com/romanresh/vscode-testcafe)
-* [SublimeText](https://github.com/churkin/testcafe-sublimetext)
+## Badge
 
-### Framework-Specific Selectors
+Show everyone you are using TestCafe: ![Tested with TestCafe](https://img.shields.io/badge/tested%20with-TestCafe-blue.svg)
 
-Work with page elements in a way that is native to your framework.
+Use the following URL to get a TestCafe badge:
 
-* [Aurelia](https://github.com/miherlosev/testcafe-aurelia-selectors)
-* [React](https://github.com/DevExpress/testcafe-react-selectors)
-* [Vue](https://github.com/devexpress/testcafe-vue-selectors)
-
-### Plugins for Task Runners
+```text
+https://img.shields.io/badge/tested%20with-TestCafe-blue.svg
+```
```

Comment:
Add the following code into your repository readme
```md
<a href="https://github.com/DevExpress/testcafe"><img alt="Tested with TestCafe" src="https://img.shields.io/badge/tested%20with-TestCafe-blue.svg"></a>    
```

Reply:
ok